src/rpc/handlers/Tx.cpp

```diff
@@ -36,6 +36,15 @@ doTx(Context const& context)
     if (!range)
         return Status{Error::rpcNOT_READY};
 
+    if (request.contains(JS(min_ledger)) && request.contains(JS(max_ledger)))
+    {
+        const auto& minLedger = request.at(JS(min_ledger));
```

Comment:
Please add a check for `is_int64()` first and then consider grabbing `as_int64()` right here - you also will not need it to be a reference that way.